_posts/2023-10-21-beneath-the-shell.md

```diff
@@ -0,0 +1,65 @@
+
+<img class="centered shadowed" src="/images/blog/ka-het/fetri-sei-replication.jpg" width="480" height="320" />
+
+> ***The four stages of Ka'sei replication***
+
+The self-replication capabilities of the Ka'sei start with the selection of a suitable asteroid to use for it; it will provide the ship with most, if not all, of the necessary materials for this activity and act as a base for the Fetri for the years to come. Once this has been done, and as long as the Ka'sei has sufficient resources, it will open the aft workstation to begin growing an organism completely identical to itself, moving only to gather additional materials as needed. In around 40 months the new organism becomes self-sufficient and can separate, first remaining connected to the original's life support through additional umbilicals and then surviving completely on its own; in fact, during this phase the older Fetri will use the forward workstation to construct the hull and mechanical systems of the new organism, a phase which takes between 5 and 7 weeks depending on material availability. Two identical Ka'sei now exist, both equally as capable and both ready to continue their mission.
+There was much internal debate during the development of the Ka'het about these systems, and whether they would be harmful or beneficial to the overall architecture; the exponential growth was considered by some to be essential to success of the fleet, while others found its size and added requirements unacceptable. In the end, the Assured Continued Production doctrine meant that the benefits were seen as irrelevant for the intended use, given that if a large part of the fleet was lost in a single event building it back would have been impossible.
+
+<img class="centered shadowed" src="/images/blog/ka-het/maeri-het-hibernated.jpg" width="480" height="360" />
+
+> ***Artist's impression of a hibernated Maeri***
+
+Early concepts called for the Maeri'het to have incredibly advanced self-piloting capabilities, returning home automatically in case the main organism was dead but the overall systems were still working, or even reactivating after it was disabled to cloak and move to a safe space. Much of this was found to be undesirable, unfeasible within the available space or straight up not within the realm of possibilities, but some traces of this confused era remained as what was called the Ka'het Hibernation System.
+During the initial time period in which the entire fleet remains stationed inside the main cluster, it is expected that a number of Maeri organisms will be lost to testing, manufacturing defects or perhaps old age; due to the high value of these vehicles, as well as the necessity of preventing the stealth system from falling into unauthorized hands, a small, inflatable device has been conceived which can maintain the Ka'het alive enough for it to be rescued and towed back to a nearby station, where it can be emptied and cheaply reused for a new one.
```

Comment:
```suggestion
Early concepts called for the Maeri'het to have incredibly advanced self-piloting capabilities, returning home automatically in case the main organism was dead but the overall systems were still working, or even reactivating after it was disabled to cloak and move to a safe space. Much of this was found to be undesirable, unfeasible within the available space, or straight up not within the realm of possibilities, but some traces of this confused era remained as what was called the Ka'het Hibernation System.
During the initial time period in which the entire fleet remains stationed inside the main cluster, it is expected that a number of Maeri organisms will be lost to testing, manufacturing defects, or perhaps old age; due to the high value of these vehicles, as well as the necessity of preventing the stealth system from falling into unauthorized hands, a small, inflatable device has been conceived which can maintain the Ka'het alive enough for it to be rescued and towed back to a nearby station, where it can be emptied and cheaply reused for a new one.
```